Bug : Last member cannot be seen in members tab while scrolling

**Describe the bug**
The last member name and details are hiding and cannot be seen while scrolling

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'Members tab in apk'
2. Scroll down 
4. See error

**Expected behavior**
The last name of the member should be clearly seen

**Screenshots**

https://user-images.githubusercontent.com/60894542/105891578-cbe1b780-6036-11eb-978a-ae98a6575803.mp4




**Smartphone (please complete the following information):**
 - Device: Redmi Y2
 - OS: Android10

Tests are not running (locally), it shows the following:
```
> Task :app:kaptDebugKotlin FAILED
e: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
        at java.base/java.lang.Class.getDeclaredConstructors0(Native Method)
...
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:kaptDebugKotlin'.
...
Caused by: org.gradle.api.GradleException: Internal compiler error. See log for more details

```

--- a/app/src/main/res/layout/activity_main.xml
+++ b/app/src/main/res/layout/activity_main.xml
@@ -53,7 +53,6 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:fillViewport="true"
-            android:layout_marginBottom="@dimen/fab_button_visible"
             app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
             <FrameLayout

--- a/app/src/main/res/layout/fragment_profile.xml
+++ b/app/src/main/res/layout/fragment_profile.xml
@@ -18,8 +18,7 @@
 
         <androidx.core.widget.NestedScrollView
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:paddingBottom="56dp">
+            android:layout_height="match_parent">
 
             <androidx.constraintlayout.widget.ConstraintLayout
                 android:layout_width="match_parent"

